README.md

```diff
@@ -1,19 +1,28 @@
-# Ore Cart App
+# Ore Cart App [![License](https://img.shields.io/badge/License-Apache_2.0-blue.svg)](https://opensource.org/licenses/Apache-2.0)
 
-[![License](https://img.shields.io/badge/License-Apache_2.0-blue.svg)](https://opensource.org/licenses/Apache-2.0)
+The master monorepo for all OreCart app software! The OreCart app includes a backend made with [FastAPI](https://fastapi.tiangolo.com) and a crossplatform mobile app made with [React-Native](https://reactnative.dev).
 
-For more details please see the Google Drive.
+For full documentation, please see our Google Drive (contributors can gain access by talking with the leads). The frontend and backend folders have READMEs with more information about developing in each as well.
 
-The frontend and backend folders have READMEs with more information about developing in each as well.
+## General Environment Setup
 
-## General Requirements
+For this project you will need [Docker](https://www.docker.com), [Node.js](https://nodejs.org/en), and [Python 3](https://www.python.org).
 
-Please use branches for each new feature and submit a pull request when ready. Do not commit to Main!
+To start up the PostgreSQL database, run `docker-compose up -d`.
```

Comment:
`docker-compose` is no longer the standard way to use docker compose.

It is now a module of the docker tool itself, so `docker compose` is the proper way to do it.